src/test/resources/application-test.properties

```diff
@@ -5,4 +5,5 @@ spring.datasource.password=
 #hibernate
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
-spring.jpa.properties.hibernate.format_sql=true
\ No newline at end of file
+spring.jpa.properties.hibernate.format_sql=true
+springdoc.swagger-ui.path=/swagger/swagger-ui.html
```

Comment:
do you use this

Reply:
Yes, I use this. **spring.jpa.properties.hibernate.format_sql=true** I use it to view requests if some tests have failed after modifications.

**springdoc.swagger-ui.path=/swagger/swagger-ui.html** 
Without this, tests cannot collect beans, because the project root references the swagger, without this the tests will not compile.